react/components/Modal/TopBar.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { forwardRef, ReactNode } from 'react'
+import classNames from 'classnames'
+
+import IconClose from '../icon/Close'
+
+export interface Props {
+  showCloseIcon?: boolean
+  showTopBar: boolean
+  children: ReactNode
+  onClose: () => unknown
+  responsiveFullScreen?: boolean
+}
+
+const TopBar = forwardRef<HTMLDivElement, Props>(function TopBar(
+  { showCloseIcon, children, onClose, responsiveFullScreen, showTopBar },
+  forwardedRef
+) {
+  const handleKeyDown = ({ key }: React.KeyboardEvent) => {
+    const SPACE = ' '
+    const ENTER = 'Enter'
+
+    if (key === SPACE || key === ENTER) {
+      onClose()
+    }
+  }
+
+  return (
+    <div className="mb5">
+      <div className="min-h-large pl8">
+        {showCloseIcon && (
+          <div
```

Comment:
You can use a `button` instead of faking the behavior of one. You may take a look at the `ButtonPlain` component as well.

Reply:
I can't get the same appearance with these components because of their hover background :/

Reply:
Done!